docs/mkdocs/docs/api/macros/nlohmann_json_namespace.md

```diff
@@ -1,27 +1,32 @@
 # NLOHMANN_JSON_NAMESPACE
 
 ```cpp
-#define NLOHMANN_JSON_NAMESPACE
+#define NLOHMANN_JSON_NAMESPACE /* value */
 ```
 
-This macro evaluates to the full name of the `nlohmann` namespace, including  the name of a versioned and ABI-tagged
-inline namespace. Use this macro to unambiguously refer to the `nlohmann` namespace.
+This macro evaluates to the full name of the `nlohmann` namespace.
 
 ## Default definition
 
-The default value consists of a prefix, a version string, and optional ABI tags depending on whether ABI-affecting
-macros are defined (e.g., [`JSON_DIAGNOSTICS`](json_diagnostics.md), and
-[`JSON_USE_LEGACY_DISCARDED_VALUE_COMPARISON`](json_use_legacy_discarded_value_comparison.md)).
+The default value consists of the root namespace, the tag namespace, and the version namespace.
 
-When the macro is not defined, the library will define it to its default value.
+- The root namespace is always `nlohmann`.
+- The tag namespace starts with `nlohmann_json` and is followed by several optional ABI tags according to the value of
```

Comment:
Why not just `json` as start of the tag namespace`

Reply:
That would conflict with the type alias `nlohmann::json`.